### DISCUSSION TITLE
Mermaid.js in docusaurus v2

### DISCUSSION THREAD

**Discussion by ghost** [facebook/docusaurus#4517] [Q&A] [answered]
Here I suppose it is about docusaurus v1: https://github.com/facebook/docusaurus/issues/1258

Is there any known way to render mermaid charts in markdown in v2 already? 

If I try the examples in [1258](https://github.com/facebook/docusaurus/issues/1258), I get `Page Not Found` without any error as soon as I `import Mermaid from '@theme/Mermaid';`

Thank you 🤗

**Comment by slorber** [comment, marked as answer]
The issue is about Docusaurus 2, particularly if you see MDX (React in markdown) and `useEffect`, it can't be Docusaurus v1.

I believe this is a good way to use Mermaid on Docusaurus: https://github.com/facebook/docusaurus/issues/1258#issuecomment-594393744

`Page Not Found` looks unrelated to me, you probably did something wrong
